cartographer/mapping/internal/3d/scan_matching/fast_correlative_scan_matcher_3d.h

```diff
@@ -115,7 +116,7 @@ class FastCorrelativeScanMatcher3D {
       const sensor::PointCloud& point_cloud,
       const Eigen::VectorXf& rotational_scan_matcher_histogram,
       const Eigen::Quaterniond& gravity_alignment, float min_score,
-      const bool log = false) const;
+      const bool full = false, const bool log = false) const;
```

Comment:
does the default argument make sense here? It seems like you set it anywhere and it has to be added everywhere anyway

Reply:
Original upstream doesn't have the parameters, is backwards compatibility with upstream to not get conflicts / have less changes.